src/kohlrahbi/unfoldedahb/unfoldedahbtable.py

```diff
@@ -286,12 +287,31 @@ def dump_flatahb_json(self, output_directory_path: Path) -> None:
 
         flatahb_output_directory_path = output_directory_path / str(edifact_format) / "flatahb"
         flatahb_output_directory_path.mkdir(parents=True, exist_ok=True)
-
-        dump_data = FlatAnwendungshandbuchSchema().dump(self.convert_to_flat_ahb())
-
-        with open(
-            flatahb_output_directory_path / f"{self.meta_data.pruefidentifikator}.json", "w", encoding="utf-8"
-        ) as file:
+        flat_ahb = self.convert_to_flat_ahb()
+
+        file_path = flatahb_output_directory_path / f"{self.meta_data.pruefidentifikator}.json"
+        if file_path.exists():
+            with open(file_path, "r", encoding="utf-8") as file:
+                existing_flat_ahb = FlatAnwendungshandbuchSchema().load(json.load(file))
+        dump_equals_existing_file_except_for_guids = flat_ahb.meta == existing_flat_ahb.meta and len(
```

Comment:
`dump_equals_existing_file_except_for_guids` I think we can improve this name.
It is not so clear at the first reading.

Can you explain what this variable controls?